src/components/map/map.tsx

```diff
@@ -17,7 +22,19 @@ import { parseCoords } from "../common/coordsValidate";
   tag: "gx-map"
 })
 export class Map implements GxComponent {
-  private map: any;
+  private map: LFMap;
+  private markersList = [];
+  private mapProviderApplied: string;
+  private mapTypesProviders = {
+    hybrid:
+      "https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}",
+    satellite:
+      "https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}",
+    standard: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+  };
+  private MIN_ZOOM = 1;
+  private RECOMMENDED_MAX_ZOOM = 20;
```

Comment:
I think we could declare this constants as `const` outside the class. What do you think?

Reply:
Done in 6eb93ae